src/EA.Weee.RequestHandlers.Tests.Unit/AatfReturn/SubmitReturnHandlerTests.cs

```diff
@@ -132,7 +123,7 @@ public async Task HandleAsync_GivenReturnThatsAlreadySubmitted_StatusUpadateMust
 
             await handler.HandleAsync(message);
 
-            A.CallTo(() => @return.UpdateSubmitted(userId.ToString())).MustNotHaveHappened();
+            A.CallTo(() => @return.UpdateSubmitted(userId.ToString(), false)).MustNotHaveHappened();
```

Comment:
Change the false to A<string>._ as we don't really mind what value that is

Reply:
Updated test